core/templates/dev/head/pages/exploration_editor/ParamChangesEditorDirective.js

```diff
@@ -83,7 +83,8 @@ oppia.directive('paramChangesEditor', [function() {
               angular.copy(DEFAULT_PARAM_SPEC));
             $scope.paramNameChoices = generateParamNameChoices();
           }
-          $scope.paramChangesService.displayed.push(newParamChange);
+          $scope.paramChangesService.displayed.push(
+            angular.copy(newParamChange));
```

Comment:
Oh ya I think I meant customizationArgs not generatorId. So I think that explains it, thanks!